### ISSUE TITLE
Help!Slideshow not working

### ISSUE THREAD

**Issue by daisyfee** [viljamis/ResponsiveSlides.js#289]
Please help! I plugged in the code given and downloaded the files but the slideshow will doesn't work when I preview in browser.
This is the code:

<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>

<script src="root folder\viljamis-ResponsiveSlides.js-1200795"></script>

<script>
  $(function() {
    $(".rslides").responsiveSlides();auto:true;speed:500;pager:true;nav:true;maxwidth:"800px";
  });
</script>

<style type="text/css">
.rslides {
  position: relative;
  list-style: none;
  overflow: hidden;
  width: 100%;
  padding: 0;
  margin: 0;
  }

.rslides li {
  -webkit-backface-visibility: hidden;
  position: absolute;
  display: none;
  width: 100%;
  left: 0;
  top: 0;
  }

.rslides li:first-child {
  position: relative;
  display: block;
  float: left;
  }

.rslides img {
  display: block;
  height: auto;
  float: left;
  width: 100%;
  border: 0;
  }
</style>

</head>

<body>

<ul class="rslides">
  <li><img src="root folder/slide1.jpg" alt=""></li>
  <li><img src="root folder/slide2.jpg" alt=""></li>
  <li><img src="root folder/slide3.jpg" alt=""></li>
</ul>

</body>
</html>

**Comment by sheriffderek** [comment]
This is a [StackOverflow](http://stackoverflow.com/) sort of question.
